examples/todo-intermediate/src/controllers/todo-list.controller.ts

```diff
@@ -1,67 +1,59 @@
-import {repository, HasManyRepository} from '@loopback/repository';
+import {Filter, Where, repository} from '@loopback/repository';
+import {post, param, get, put, patch, del, requestBody} from '@loopback/rest';
+import {TodoList} from '../models';
 import {TodoListRepository} from '../repositories';
-import {TodoList, Todo} from '../models';
-import {requestBody, param, post, get, del, put, patch} from '@loopback/rest';
 
 export class TodoListController {
-  protected todoRepo: HasManyRepository<TodoList>;
-
   constructor(
-    @repository(TodoListRepository) protected todoListRepo: TodoListRepository,
+    @repository(TodoListRepository)
+    public todoListRepository: TodoListRepository,
   ) {}
 
   @post('/todo-lists')
-  async createTodoList() {
-    const newTodoList = {createdAt: new Date()};
-    return await this.todoListRepo.create(newTodoList);
+  async create(@requestBody() obj: TodoList): Promise<TodoList> {
```

Comment:
Note, the controller has been refactored with the one that can be generated through the CLI

Reply:
Should we preserve the feature that sets `createdAt` to the current date?

I think this feature should be implemented at model level via `defaultFn: 'now'` setting. 

```ts
@property({name: 'createdAt', type: Date, defaultFn: 'now'});
```